charts/aws-ebs-csi-driver/templates/controller.yaml

```
@@ -212,7 +212,7 @@ spec:
         {{- end }}
         - name: csi-resizer
           image: {{ printf "%s:%s" .Values.sidecars.resizer.image.repository .Values.sidecars.resizer.image.tag }}
-          imagePullPolicy: Always
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
```

I would prefer to see this as `{{ .Values.sidecars.resizer.pullPolicy }}` or
```suggestion
          imagePullPolicy: {{ default .Values.image.pullPolicy .Values.sidecars.resizer.pullPolicy }}
```

For consistency, I would also like to see the option added for the other sidecars.  Also, don't forget to update the values.yaml for the new sidecar values.

---

Updated.